core/src/main/java/io/crate/Version.java

```diff
@@ -40,8 +40,8 @@
     // the (internal) format of the id is there so we can easily do after/before checks on the id
 
 
-    public static final boolean SNAPSHOT = false;
-    public static final Version CURRENT = new Version(3010699, SNAPSHOT, org.elasticsearch.Version.CURRENT);
+    public static final boolean SNAPSHOT = true;
```

Comment:
why was `SNAPSHOT` changed from false -> true?

Reply:
Because, after the 3.1.6 release, we can now say that the state of the 3.1 branch is a SNAPSHOT of an upcoming 3.1.7 change, for when we build nightlies.